GOVERNANCE.md

```diff
@@ -0,0 +1,54 @@
+| Modeling Expert | Modeling experts are knowledgeable about the modeling approach based on BAMM and have an overview of other already existing models. |
+| Model Developers | Group of Domain and Modeling experts who jointly perform a modeling activity (create, update or delete model) |
+| Modeling Team | Regular meeting round to discuss and moderate the modeling activities. |
+| Use Case | Application or scenario were the need for the model creation or update arises. In general, it is assumed that a use case consists of at least one data provider and one data consumer |
+
+### MS1 Request for Model
+
+When there is a requirement for a new or modified Aspect model, a domain expert from that use case initiates the modeling process by creating a new issue on GitHub (Tab: Issues -> Button: New Issue or [this link](https://github.com/eclipse-tractusx/sldt-semantic-models/issues/new/choose)). 
+Depending on the type of request (new model, model update, model deprecation), one may use different templates for the issue. 
+Based on the information given in the issue description, the modeling team decides whether to approve and to progress with the development of this model. 
+In that case, the modeling team assigns the label "MS1-Approved" and names at least one modeling expert in the GitHub issue who will support the domain experts in resolving the issue.
+
+With the MS1 checklist and approval, the idea is to initiate the exchange between domain and modeling experts in an early development phase.
+This way, model developers get fast feedback before investing time and effort on models for which they might not get an agreement within the modeling team. 
+The discussion of the model modifications shall further create awareness of currently ongoing modeling activities across modelers and use cases to avoid duplicate models and development for the same topic.
+
+### MS2 Valid Model
+In the next step, the assigned domain experts and the modeling experts perform the modeling work and create a solution to fulfill the initial request.
+This work happens on a dedicated development branch preferably in a forked repository (for details regarding Forks in GitHub consult the [GitHub documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks)).
+As a result of this phase, the model developers propose the changes by creating a Pull-Request (PR) to commit the content of the new or updated model to the main-branch of the Tractus-X model repository.
+To create a PR, select "New pull request" in the "Pull requests" tab.
+The PR should link the initial issue requesting the change, e.g., by mentioning #{Issue-Number} in the description.
+There is a template for this PR, which contains a checklist of modeling-specific guidelines.
```

Comment:
should we adjust the checklist as well. EG the IP check is done via ECA. Maybe there is more we need to adjust

Reply:
yes, for the checklist we will have a dedicated PR